dasharo-security/secure-boot.robot

```diff
@@ -252,137 +250,129 @@ SBO009.001 Attempt to boot file signed for intermediate certificate
     ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
     Enter Enroll DB Signature Using File In DB Options    ${advanced_menu}
     Enter Volume In File Explorer    NO VOLUME LABEL
-    Select File In File Explorer    intermediate-db.der
+    Select File In File Explorer    cert.der
     Save Changes And Reset    3    5
     Enter UEFI Shell
     ${out}=    Execute File In UEFI Shell    hello.efi
     Should Contain    ${out}    Access Denied
     ${out}=    Execute File In UEFI Shell    signed-hello.efi
+    Should Contain    ${out}    Hello, world!
 
 SBO010.001 Check support for rsa2k signed certificates
     [Documentation]    PEM generated with `openssl req -new -x509 -newkey rsa:2048 -subj "/CN=DB-RSA2048/" -keyout DB-RSA2048.key -out DB-RSA2048.crt -days 3650 -nodes -sha256`
     ...    converted to DER using `openssl base64 -d -in DB-RSA2048.cer -out DB-RSA2048.der`
-    Download ISO And Mount As USB    ${DL_CACHE_DIR}/${RSA2_K_TEST_NAME}    ${RSA2_K_TEST_URL}    ${RSA2_K_TEST_SHA256}
+    Mount ISO As USB    RSA2048
     Power On
 
     ${sb_menu}=    Enter Secure Boot Menu And Return Construction
     Enable Secure Boot    ${sb_menu}
     ${sb_menu}=    Reenter Menu And Return Construction
     ${advanced_menu}=    Enter Advanced Secure Boot Keys Management And Return Construction    ${sb_menu}
     Enter Enroll DB Signature Using File In DB Options    ${advanced_menu}
-    Enter Volume In File Explorer    RSA2K_LABEL
-    Select File In File Explorer    DB-RSA2048.der
+    Enter Volume In File Explorer    NO VOLUME LABEL
```

Comment:
@WiktorG351 I think @m-iwanicki in his latest PR sets some labels for this images, maybe let's use them?

Reply:
@TomaszAIR adressed, i added labels in [this commit](https://github.com/Dasharo/open-source-firmware-validation/pull/192/commits/4de28123ddcbff4af3746e7b54496cdab3c7163f)